main.go

```diff
@@ -9,6 +9,11 @@ import (
 	"regexp"
 	"time"
 
+	"encoding/base64"
```

Comment:
Standard library dependencies are separated from external dependencies. `encoding/base64` should be above the empty line.